src/braket/schema_common/schema_base.py

```diff
@@ -52,7 +52,13 @@ def import_schema_module(schema: BraketSchemaBase):
         name = schema.braketSchemaHeader.name
         version = schema.braketSchemaHeader.version
         module_name = name + "_v" + version
-        return import_module(module_name)
+        try:
+            return import_module(module_name)
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(
+                f"Amazon Braket could not find the module, f{module_name}. "
+                "To continue, upgrade your installation of amazon-braket-schemas."
```

Comment:
Since this is customer-facing verbiage, you may want to double check it with Leslie (@lundql)

Reply:
Already did! This was her iteration on the error string